build.override.targets

```diff
@@ -55,4 +57,16 @@
     <Message Text="$([System.IO.File]::ReadAllText($(WcfSelfHostCleanupLog)))" Condition="Exists('$(WcfSelfHostCleanupLog)')" Importance="High" />
     <Message Text="-------------- WCF clean end --------------" Importance="High" />
   </Target>
+
+  <!-- This target will copy a master version of project.json that uses last released version of non-WCF dependencies to the scenario test dirs. -->
+  <!-- These non-WCF dependencies "may" explicitly reference a pre-release version but should only do so to pull in needed fixes. -->
+  <Target Name="WcfCopyProjectJson" Condition="'$(UseStableDependencies)'=='true'">
```

Comment:
It's my understanding this target should be unconditioned. Inside the target, however, we will copy different project.json files depending on this condition. Is this the idea?